joomla/joomla-cms
# [4.0] Move changelog JS to it's own file

Pull Request for Issue #24849

Here goes my first try at some modular JS

### Summary of Changes
Fixed the missing modal IDs in the link
Moved all changelog related JavaScript to it's own file

### Testing Instructions
1. Apply this pull request
2. Run npm run build:js
3. Install the test extension from https://github.com/roland-d/component_joomla/releases/tag/v0.0.2
4. Go to Manage
5. Filter the list on List of Students
6. Click on the version number 0.0.2
7. A modal shows up with the changelog as shown below under Manage View
8. Go to Update
9. Click on Find Updates
10. The update is shown and click on the Changelog button
11. A modal shows up with the changelog as shown below under Update View


### Expected result
There is no JavaScript error in Update view
Specific changelog JavaScript is used

### Actual result
There is a JavaScript error in Update view
Core Joomla JavaScript is used

### Documentation Changes Required
None

Please review:
@euismod2336 @C-Lodder
